### PR TITLE
chore(views): clean up module component view

### PR DESCRIPTION
**Pull request by hypeJunction**
Removes inline html and allows class to be an array
- [x] fix https://github.com/Elgg/Elgg/pull/8232/files#r29054364

### CODE REVIEW

File: views/default/page/components/module.php [hunk 1]
```diff
@@ -11,43 +12,39 @@
...
-} elseif ($title) {
-	$header = "<div class=\"elgg-head\"><h3>$title</h3></div>";
+	$header = elgg_extract('header', $vars, '');
+} else if ($title) {
+	$header = elgg_format_element('h3', [], $title);
 }
 
-$body = "<div class=\"elgg-body\">$body</div>";
-
+if (isset($header)) {
```
Comment (mrclay) [inline views/default/page/components/module.php]: This looks right. Only thing bugs me is that this relies on logic where a variable may or may not be present, and I think that invites errors. I'd probably prefer setting it to null if missing and explicitly checking null later.

Comment (mrclay) [inline views/default/page/components/module.php]:
```php
$header = elgg_extract('header', $vars);
if ($title) {
    $header = ...
}

if ($header !== null) {
    ...
}
```

File: views/default/page/components/module.php [hunk 2]
```diff
@@ -11,43 +12,39 @@
...
+	$header = elgg_format_element('div', ['class' => 'elgg-head'], $header);
+}
+$body = elgg_format_element('div', ['class' => 'elgg-body'], $body);
 if ($footer) {
-	$footer = "<div class=\"elgg-foot\">$footer</div>";
+	$header = elgg_format_element('div', ['class' => 'elgg-head'], $header);
```
Comment (jdalsem) [inline views/default/page/components/module.php]: shouldn't this be $footer and a footer class?

Comment (mrclay) [inline views/default/page/components/module.php]: It does look that way

**Comment by hypeJunction** [comment]
@mrclay @jdalsem Comments have been addressed

**Pull request merged**